packages/jest-config/file-transform.js

```diff
@@ -0,0 +1,7 @@
+const last = (arr) => arr[arr.length - 1];
+
+module.exports = {
+  process(src, filename) {
+    return `module.exports = ${JSON.stringify(last(filename.split("/")))};`;
```

Comment:
mb native api?
```
const path = require("path");
path.basename("hello/world.js")
```

Reply:
Ye, this is also an option - I mostly did this to avoid ESlint complaints:
```
Require statement not part of import statement.eslint[@typescript-eslint/no-var-requires](https://typescript-eslint.io/rules/no-var-requires)
```

Happy to use `require` and suppress the error though if you prefer that

Reply:
I just have no idea how it will work for windows users, or do we support them ;-)

Reply:
I think that this might come here already normalized - but ye, I'm not 100% sure about this.

Reply:
to avoid eslint complaints we can disable it for the line)))

Reply:
ok, i went that way